lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -273,6 +273,7 @@ def __init__(self):
         self.election_tick = None
         self.dispatcher_heartbeat_period = None
         self.node_cert_expiry = None
+        self.node_id = None
```

Comment:
Please remove this line. `TaskParameters` is only for parameters which belong to the docker swarm state, not to operations. Everything in here is used for idempotency checking, and `node_id` should not be used for that :)

Reply:
Ah thanks, didn't know that.

Reply:
No problem, it's also rather new (at most a couple of weeks old) :)